tests/core/test_views_errors.py

```diff
@@ -197,6 +197,16 @@ def test_500_passes_exception_as_exc_info(self):
         self.assertEqual(mocked_logger.error.call_args[1]['exc_info'], custom_error)
 
 
+class ErrorPostgresViewTest(FormattedErrorMixin, PostgreSQLTest, unittest.TestCase):
```

Comment:
Since the fix happens at the resource level I would prefer not to involve PostgreSQL here.

Could you reproduce the original issue with that test?

Reply:
Yes, when testing against Postgre I get the original error that you posted in the issue. If I just use the ```BaseWebTest``` then I was unable to reproduce the error.

Reply:
Ok I see.

The thing is that now we have a safety check / fix (and some tests that assert the proper behavior) at the resource level. We don't have to imply PostgreSQL anymore. `BaseWebTest` should be enough :wink: